<ui version="4.0" >
 <author>Tomeu Borras</author>
 <class>ContratoBase</class>
 <widget class="QWidget" name="ContratoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Nuevo Contrato</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >
    <normaloff>:/Bulmafact48x48/images/png/48x48/i_bulmages_documentos.48x48.png</normaloff>:/Bulmafact48x48/images/png/48x48/i_bulmages_documentos.48x48.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="pixmapLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_documentos.48x48.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>5</number>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar Contrato</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar Contrato</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/edit-save.svg</normaloff>:/Images/edit-save.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar Contrato</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar Contrato</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff>:Images/edit-delete.svg</normaloff>:Images/edit-delete.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_facturar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Facturar Periodos Pendientes</string>
          </property>
          <property name="whatsThis" >
           <string>Facturar Periodos Pendientes</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff>:/Bulmafact32x32/images/png/32x32/i_bulmages_ventas_facturas_automaticas.32x32.png</normaloff>:/Bulmafact32x32/images/png/32x32/i_bulmages_ventas_facturas_automaticas.32x32.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>16</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="mui_plugbotones" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>81</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label5" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Referencia:</string>
          </property>
          <property name="buddy" >
           <cstring>mui_refcontrato</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BusquedaReferencia" name="mui_refcontrato" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Ignored" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="mui_detalles" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Mostrar/ocultar cabecera</string>
       </property>
       <property name="whatsThis" >
        <string>Mostrar/ocultar cabecera</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../bulmages.qrc" >
         <normaloff>:/Images/document-open.svg</normaloff>:/Images/document-open.svg</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label7" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Descripcion:</string>
       </property>
       <property name="buddy" >
        <cstring>mui_nomcontrato</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mui_nomcontrato" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>50</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item rowspan="3" row="0" column="0" >
       <widget class="BlSearchWidget" native="1" name="mui_idcliente" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Ignored" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>250</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label8_3" >
          <property name="font" >
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Periodicidad Facturas:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>mui_periodicidadcontrato</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlPeriodicityComboBox" name="mui_periodicidadcontrato" />
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Ubicacion:</string>
          </property>
          <property name="buddy" >
           <cstring>mui_loccontrato</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mui_loccontrato" />
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label8_2" >
          <property name="font" >
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Fecha de finalizacion:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlDateSearch" native="1" name="mui_ffincontrato" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Ignored" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label8" >
          <property name="font" >
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Fecha de creacion:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlDateSearch" native="1" name="mui_fincontrato" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Ignored" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="Detalle" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>676</width>
        <height>179</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Detalle</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="ListLinContratoView" name="mui_lineas" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>409</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Facturas</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="FacturasContratoListSubForm" native="1" name="subform2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>409</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Comentarios</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QTextEdit" name="mui_descontrato" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>436</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <customwidgets>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QWidget</extends>
   <header>../../../bulmalib/src/bldatesearch.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaReferencia</class>
   <extends>QLineEdit</extends>
   <header>busquedareferencia.h</header>
  </customwidget>
  <customwidget>
   <class>FacturasContratoListSubForm</class>
   <extends>QWidget</extends>
   <header>contratoview.h</header>
  </customwidget>
  <customwidget>
   <class>ListLinContratoView</class>
   <extends>QTableWidget</extends>
   <header>listlincontratoview.h</header>
  </customwidget>
  <customwidget>
   <class>BlPeriodicityComboBox</class>
   <extends>QComboBox</extends>
   <header>blperiodicitycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Busqueda</class>
   <extends>QWidget</extends>
   <header>blsearchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_detalles</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>27</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>85</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
